tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json

```diff
@@ -0,0 +1,32 @@
+{
+    "symbol": "RMESH",
+    "address": "0x8d5682941ce456900b12d47ac06a88b47c764ce1",
+    "decimals": 18,
+    "name": "RightMesh Token",
+    "ens_address": "",
+    "website": "https://www.rightmesh.io/",
+    "logo": {
+      "src": "https://avatars2.githubusercontent.com/u/29805586?s=128",
+      "width": "92",
+      "height": "92",
+      "ipfs_hash": "QmQ9tzSaUpjrhsZ6vFjp8P5FcBWFEKrYD8joi6d8HVXcdc"
+    },
+    "support": {
+      "email": "contact@rightmesh.io"
```

Comment:
not sure if contact@rightmesh.io goes anywhere - I emailed it today to see if anyone answered and got nothing back.

I do know that tech@rightmesh.io will go to me and the dev team so maybe use that.